coffee_plugin/src/plugin/plugin_mod.rs

```diff
@@ -98,7 +98,7 @@ impl RPCCommand<State> for CoffeeInstall {
         let rt = Runtime::new().unwrap();
 
         let request: InstallReq = serde_json::from_value(request)?;
-        rt.block_on(coffee.install(&request.name, false, true))
+        rt.block_on(coffee.install(&request.name, false, true, None))
```

Comment:
```suggestion
        rt.block_on(coffee.install(&request.name, false, true, Some(request.branch)))
```

if the branch do not exist please add it to the `InstallReq `